README.md

```diff
@@ -170,6 +170,9 @@ In JSON mode, each job posting is a JSON object with the following fields:
 | hostedUrl   | A URL which points to Lever's hosted job posting page. Examples: [global][leverdemo-job-site-posting-global] / [EU][leverdemo-job-site-posting-eu]
 | applyUrl    | A URL which points to Lever's hosted application form to apply to the job posting. Examples: [global][leverdemo-job-site-posting-application-global] / [EU][leverdemo-job-site-posting-application-eu]
 | workplaceType    | Describes the primary workplace environment for a job posting. May be one of `unspecified`, `on-site`, `remote`, or `hybrid`. Not filterable. <br /> Note: to be released in waved rollouts starting October, 2022.
+| salaryRange  | Object with `currency`, `interval`, `min`, and `max`.  This field is optional.  In XML mode this field is parsed into a string.<br /> Note: to be released in waved rollouts starting mid February, 2023.
```

Comment:
I wonder if we need to specify that in case that some of the attributes of this object is filled, the object itself turns into a **required** field. e.g: If the `min` attribute is populated the rest of the attributes should be populated as well.
cc. @mattngan

Reply:
so since this is a read only API I dont think we need to relay this.  They wont be filling out these values using this API